packages/gatsby-plugin-gatsby-cloud/src/components/InfoIndicatorButton.js

```diff
@@ -45,16 +46,33 @@ const getButtonProps = ({ status, createdAt }) => {
   }
 }
 
-export default function InfoIndicatorButton({ status, createdAt }) {
+export default function InfoIndicatorButton({
+  status,
+  createdAt,
+  buildId,
+  orgId,
+  siteId,
+}) {
   const buttonProps = getButtonProps({ status, createdAt })
 
+  const trackHover = () => {
+    trackEvent({
+      eventType: `PREVIEW_INDICATOR_HOVER`,
+      orgId,
+      siteId,
+      buildId,
+      name: `info hover`,
+    })
+  }
+
   return (
     <IndicatorButton
       testId="info"
       iconSvg={infoIcon}
       {...buttonProps}
       // See IndicatorButtonTooltip for explanation
       toolTipOffset={80}
+      mouseOverCallback={buttonProps.active && trackHover}
```

Comment:
Events are mostly starting with "on"

```suggestion
      onMouseOver={buttonProps.active && trackHover}
```